commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/impl/PacketReaderImpl.java

```diff
@@ -147,7 +147,7 @@ public Map<String, Object> getAll(String id, String source, String process) {
                         if (value != null && (value instanceof Number))
                             finalMap.putIfAbsent(key, value);
                         else if (value != null && (value instanceof String))
-                            finalMap.putIfAbsent(key, value.toString().replaceAll("(^\"|\"$)", ""));
+                            finalMap.putIfAbsent(key, value.toString().replaceAll("(^\")|(\"$)", ""));
```

Comment:
Please test the api with changes to avoid issue in dev env

Reply:
tested locally